lib/include-code-extension.js

```diff
@@ -19,7 +19,9 @@ function createExtensionGroup (context) {
       this.named('include-code')
       this.process((parent, target, attrs) => {
         const doc = parent.getDocument()
-        const includeCodeUsePath = doc.getAttribute('include-code-relative-option') !== undefined
+        const realtivePathPrefix = './'
+        const useRelativePath = target && target.startsWith(realtivePathPrefix)
+        const sanitizedTarget = useRelativePath ? target.substring(realtivePathPrefix.length, target.length) : target
```

Comment:
substring is deprecated. Use slice instead. And slice does not require specifying the end. So this can be simplified to:

```
target.slice(relativePathPrefix.length)
```